### PR TITLE
[patch] fix node 14 warnings by upgrading prompt dependency

### PR DESCRIPTION
**Pull request by DominusKelvin**

### CODE REVIEW

**Comment by eashaw** [comment]
Hi @DominusKelvin, Would you be able to describe the bug this is fixing in more detail? I want to make sure I'm looking for the right thing when I test this out. Thanks for the PR!

**Comment by DominusKelvin** [comment]
> Hi @DominusKelvin, Would you be able to describe the bug this is fixing in more detail? I want to make sure I'm looking for the right thing when I test this out. Thanks for the PR!

@eashaw if you run sails lift on Node.js 14 and above you get a warning about circular dependency. This patch fixes that.

**Comment by DominusKelvin** [comment]
@eashaw here is a Loom video showing the reason why for this Patch
https://www.loom.com/share/afa1f768f29a4847a15b30656e1a4e5b

**Comment by DominusKelvin** [comment]
@eashaw any word on this yet?

**Review by eashaw — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "parseurl": "1.3.2",
     "path-to-regexp": "1.5.3",
     "pluralize": "1.2.1",
-    "prompt": "0.2.14",
+    "prompt": "^1.1.0",
```
Comment (mikermcneil) [inline package.json:57]: Since `prompt` is an externally maintained dep, lets use a strict semver range `1.1.0`

**Review by mikermcneil — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (DominusKelvin) [reply package.json]: Okay that works well

**Pull request merged**